hmrc/amls-frontend
# AMLS-4170 | HVD Page split

Pull request includes the page split and related fixes - see [comments on ticket](https://jira.tools.tax.service.gov.uk/browse/AMLS-4170) for information around making all questions editable, and Date of Change redirects.

## How Has This Been Tested?

Manual testing, spec tests.
Manual testing included:

- Changing from yes -> no
- Changing from no -> yes and answering second question
- Changing from no -> yes and not answering second question (should see incomplete section), then re-entering section and completing.
- Date of Change redirects correctly on _Products_, _HowToSellGoods_, and _Excise Goods_ pages (see description on [ticket](https://jira.tools.tax.service.gov.uk/browse/AMLS-4170)).

Acceptance tests will need to be updated and run.

## Checklist




- [x] Unit tests have full coverage.
- [ ] Unit test approach and implementation has been reviewed with QA (testers).
- [x] Requires acceptance test run.
- [x] Appropriate labels added.
- [x] RELEASE NOTES ADDED.
